### ISSUE TITLE
[med] : All Delete popup: Chrome + Jaws: Focus order is not sequential on 'Delete dialogue'.

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8525]
## Action Performed:
1. Using Chrome + Jaws, open URL staging.new.expensify.com
2. Navigate to Setting using Tab key and press Enter to select it.
3. Navigate to any workspace using Tab key and press using Enter to select it.
4. Navigate to More button using Tab key and press Enter to activate it.
5. Navigate to 'Delete workspace' control using Tab key and press Enter to activate.
6. Observe the focus order on the dialoged.

## Expected Result:
Focus should first land on the 'Delete' control after the 'Cancel' button.

## Actual Result:
After open the dialogue first focus lands on the Cancel control then it's move to back screen, after it lands on the delete. 

**Other Occurrences:**
>Same issue repro on PR #8712 -Incorrect focus order after pressing the setting button.
>Same issue repro on PR #8725 -Incorrect focus order after creating new room.
>Same issue repro on PR #8827 -After log-in, first focus start from the chat box.
>Same issue repro on PR #8969 -Incorrect focus order after closing the new chat floating button expanded content.

## Workaround:
Yes

## Area issue was found in:
Delete workspace

## Failed WCAG checkpoints
3.4.3

## User impact:
When keyboard focusable components do not receive focus in a logical order, people with mobility impairments, reading disabilities, and low vision are all impacted

## Suggested resolution:
Fix this issue by ensuring the DOM order is logical so that the focus order will naturally follow.

Refer:
http://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G59

## Platform:
- Web
- Desktop App

**Version Number: **v1.1.49-1
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:** Stuti

https://user-images.githubusercontent.com/97600950/162145803-96650ef3-5018-4b84-add2-8d1d991cda71.mp4
